src/plugins/mf_priority.cpp

```diff
@@ -204,6 +216,74 @@ static int validate_cb (flux_plugin_t *p,
                                     "user does not belong to specified bank");
     }
 
+    mj_it = users_mj.find (userid);
+
+    if (bank == NULL) {
+        def_bank_it = users_def_bank.find (userid);
+        mj_inner_it = mj_it->second.find (def_bank_it->second);
+
+        if (flux_jobtap_job_aux_set (p,
+                                     FLUX_JOBTAP_CURRENT_JOB,
+                                     "mf_priority:bank",
+                                     &def_bank_it->second[0],
+                                     NULL) < 0)
+            flux_log_error (h, "flux_jobtap_job_aux_set");
+    } else {
+        mj_inner_it = mj_it->second.find (bank);
+
+        if (flux_jobtap_job_aux_set (p,
+                                     FLUX_JOBTAP_CURRENT_JOB,
+                                     "mf_priority:bank",
+                                     &bank[0],
+                                     NULL) < 0)
+            flux_log_error (h, "flux_jobtap_job_aux_set");
+    }
```

Comment:
Suggestion: It might be cleaner to refactor the bank lookup, if necessary, into a separate function. Then you can do something like:
```c
if (!bank
    && !(bank = lookup_default_bank (user))
    return flux_jobtap_reject_job (p, args, "unable to find default bank");

// proceed with "bank" as usual...

```